### DISCUSSION TITLE
madcad.show() how to change backgroundcolor?

### DISCUSSION THREAD

**Discussion by GlennWSo** [jimy-byerley/pymadcad#53] [Q&A] [answered]
How do change the background color when calling show() ?

As i run a darkmode in my text editor find the default white background to be blinding.

Maybe something is wrong with my system. 
### Example:
```python
import madcad as cad
nut = cad.nut(5)
cad.show([nut])
```
#### My Result:
![white_nut](https://user-images.githubusercontent.com/107837123/207873293-097c9ef5-17b8-4a26-9259-c4e42878d370.png)

#### What I want:

![nice colors](https://madcad.netlify.app/screenshots/show-bearing.png)

**Comment by jimy-byerley** [comment, marked as answer]
Welcome here :)
The default color mode is the system color as seen from Qt, meaning its the system color when using the KDE desktop environment on linux (dark in my case) or any Qt based desktop, and white everywhere else.

You can customize the colors by either
- disabling the system color theme and let madcad use its own colors
  ```python
  cad.settings.display['systeme_theme'] = False
  ```
  Or changing the matching value in `~/.config/madcad/pymadcad.yaml`
- changing the color palette of Qt  (this is in case you're making an application integrating madcad)

**Reply by jimy-byerley** [reply]
It's perfectly true that light theme is blinding on top a dark environment, event writing this I'm using github's own dark mode and it's pretty blinding ;)

**Reply by GlennWSo** [reply]
Thx!
Now i can take off my sunglases 8-)

`cad.settings.display['system_theme'] = False` works like a charm!
I will use the file [madcad/*settings.py*](https://github.com/jimy-byerley/pymadcad/blob/master/madcad/settings.py) for reference in the future.

**Reply by GlennWSo** [reply]
is there a sample pymadcad.yaml somewhere that i can use as a base in my ~/.config/madcad/pymadcad.yaml ?

**Reply by jimy-byerley** [reply]
Nice ! 
Also for the config file: by default `madcad.settings` doesn't create the config file. But you can ask to create it using `madcad.settings.install()` and modify the content of the config as you want. the config is loaded when madcad is imported the first time in the program.

**Reply by jimy-byerley** [reply]
`madcad.settings.install()` does create the config file using the current settings in `madcad.settings`, which are the base settings if not altered by the program

**Reply by GlennWSo** [reply]
Awesome! 😊
